concepts/Common/Integration Guides/50 Use MUI components in your DevExtreme application/50 Use MUI components in your DevExtreme application.md

```diff
@@ -0,0 +1,182 @@
+# Use MUI components in DevExtreme apps
+
+MUI is a popular line of React components and design tools. The design of MUI components follows Google's Material Design guidelines. Most MUI products are open-source, and do not require a commercial license.
+
+You don't have to use MUI to add a Material Design look and feel to your application. The DevExtreme library includes [a range of  Material Design themes](https://devexpress.github.io/ThemeBuilder/master/material/blue-light). However, if you want to use both MUI components and DevExtreme components, you can.
+
+You don't need extra configuration to run MUI components side by side with DevExtreme components. But if you nest components from different frameworks, you have to understand how they interact with one another.
+
+## Run the sample application
+
+The DevExtreme team created a single-page, ready-to-run Vite application that illustrates two common use cases. It contains an MUI component with DevExtreme children, and a DevExtreme component with MUI children. You can view the application's source code in the following GitHub repository: [https://github.com/DevExpress-Examples/devextreme-mui-integration](https://github.com/DevExpress-Examples/devextreme-mui-integration).
```

Comment:
```suggestion
The DevExtreme team created a single-page, ready-to-run Vite application that illustrates two common use cases. It contains an MUI component with DevExtreme children, and a DevExtreme component with MUI children. You can view the application's source code in the following GitHub repository: [https://github.com/DevExpress-Examples/devextreme-mui-integration](https://github.com/DevExpress-Examples/devextreme-mui-integration).
```

Add a button with #includes instead of a link